Assets/scss/base/_var.scss

```diff
@@ -49,14 +48,11 @@ $font-normal:normal;
     --black:#{$black};
     --darkestgrey: #352f2f;
     --warm-grey : #edeae4;
-    --dark-blue: #353435;
 
     --banner-bg-color: var(--darkestgrey);
     --banner-text-color: var(--white);
-
-    --hero-banner-bg-color: var(--warm-grey);
-    --hero-banner-text-color: var(--black);
-
+    --card-bg-color: var(--white);
+    --card--title-color: var($darkestgrey);
```

Comment:
This will compile the SCSS into `var(#353435)` which might not be valid? Could simplify to `--card--title-color: $darketsgrey;`

Reply:
Updated